.internal-ci/helm/fog-ingest/templates/_helpers.tpl

```diff
@@ -24,50 +24,33 @@ If release name contains chart name it will be used as a full name.
 {{- end }}
 
 {{/* Create chart name and version as used by the chart label. */}}
-{{- define "fogIngest.chart" -}}
+{{- define "fog-ingest.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" | trimSuffix "." }}
 {{- end }}
 
 {{/* Common labels */}}
-{{- define "fogIngest.labels" -}}
-helm.sh/chart: {{ include "fogIngest.chart" . }}
-{{ include "fogIngest.selectorLabels" . }}
+{{- define "fog-ingest.labels" -}}
+helm.sh/chart: {{ include "fog-ingest.chart" . }}
+{{ include "fog-ingest.selectorLabels" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end }}
 
 {{/* Selector labels */}}
-{{- define "fogIngest.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "fogIngest.name" . }}
+{{- define "fog-ingest.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "fog-ingest.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
 
 {{/* Generate fog-ingest Peers List */}}
-{{- define "fogIngest.peerURLs" }}
+{{- define "fog-ingest.peerURLs" }}
   {{- $peerURLs := list }}
-  {{- $name := include "fogIngest.fullname" . }}
+  {{- $name := include "fog-ingest.fullname" . }}
   {{- $namespace := .Release.Namespace }}
   {{- range $i, $e := until (int .Values.fogIngest.replicaCount ) }}
     {{- $peerURLs = append $peerURLs (printf "insecure-igp://%s-%d.%s.%s.svc.cluster.local:8090" $name $i $name $namespace) }}
   {{- end }}
   {{- join "," $peerURLs }}
 {{- end }}
-
-{{/* Mobilecoin Network monitoring labels */}}
```

Comment:
remove `lookup` functions the helm project will probably never fix the usability bug around lookups, dry-run and template actions.